internal/cypress/sauce/runner.go

```diff
@@ -78,31 +86,53 @@ func (r *Runner) runSuites(fileID string) int {
 
 	// Collect results.
 	errCount := 0
-	for i := 0; i < len(r.Project.Suites); i++ {
-		r := <-results
-		if r.err != nil {
-			log.Err(r.err).Str("suite", r.suiteName).Msg("Suite failed.")
+	completed := 0
+	total := len(r.Project.Suites)
+	inprogress := total
+	passed := true
+
+	log.Info().Msgf("Suites completed: %d in progress: %d", completed, inprogress)
+	for i := 0; i < total; i++ {
+		res := <-results
+		// in case one of test suites not passed
+		if !res.job.Passed {
+			passed = false
+		}
+		completed++
+		inprogress--
+		logSuite(completed, inprogress, res.suiteName, res.browser, res.job.Passed)
+
+		if res.err != nil {
+			assetContent, err := r.JobReader.GetJobAssetFileContent(context.Background(), res.job.ID, resto.ConsoleLogAsset)
+			if err != nil {
+				log.Warn().Str("suite", res.suiteName).Msg("Failed to get job asset.")
```

Comment:
Maybe we should add the asset filename that we are trying to get ?

Reply:
Thanks!